qiskit_braket_provider/providers/adapter.py

```diff
@@ -416,6 +416,11 @@ def convert_qiskit_to_braket_circuit(circuit: QuantumCircuit) -> Circuit:
     Returns:
         Circuit: Braket circuit
     """
+    if not isinstance(circuit, QuantumCircuit):
+        raise TypeError(
+            f"Expected a qiskit.QuantumCircuit, got {type(circuit)} instead"
```

Comment:
Minor nit: this does line up with the type hint. If others feel this is more clear than just `QuantumCircuit`, I am fine with keeping it.

Reply:
Either works for me, happy to change it lmk